src/ensembl/src/shared/components/step/Step.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import styles from './Step.scss';
+
+type StepProps = {
+  count: number;
+  title: string;
```

Comment:
change title to label

Reply:
Create moreInfo react node and pass to Step component which then renders it

Reply:
Let's do it in a separate PR; looks like a chunk of work that can be cleanly separated from this one.